code/datums/station_traits/neutral_traits.dm

```diff
@@ -385,3 +385,124 @@
 	if(!pure)
 		for(var/obj/effect/landmark/start/ai/secondary/secondary_ai_spawn in GLOB.start_landmarks_list)
 			secondary_ai_spawn.latejoin_active = TRUE
+
+#define SKUB_IDFC "i don't frikkin' care"
+
+/// A trait that lets players choose whether they want a skub and pro-skub shirt, an anti-skub shirt, or neither.
+/datum/station_trait/skub
+	name = "The Great Skub Contention"
+	trait_type = STATION_TRAIT_NEUTRAL
+	show_in_report = FALSE
+	weight = 2
+	sign_up_button = TRUE
+	/// List of people signed up to be either pro_skub or anti_skub
+	var/list/skubbers = list()
+
+/datum/station_trait/skub/New()
+	. = ..()
+	RegisterSignal(SSdcs, COMSIG_GLOB_JOB_AFTER_SPAWN, PROC_REF(on_job_after_spawn))
+
+/datum/station_trait/skub/setup_lobby_button(atom/movable/screen/lobby/button/sign_up/lobby_button)
+	RegisterSignal(lobby_button, COMSIG_ATOM_UPDATE_OVERLAYS, PROC_REF(on_lobby_button_update_overlays))
+	lobby_button.desc = "Are you pro-skub or anti-skub? Click to cycle through pro-skub, anti-skub, random and neutral."
+	return ..()
+
+/// Let late-joiners jump on this gimmick too.
+/datum/station_trait/skub/can_display_lobby_button(client/player)
+	return sign_up_button
+
+/// We don't destroy buttons on round start for those who are still in the lobby.
+/datum/station_trait/skub/on_round_start()
+	return
+
+/datum/station_trait/skub/on_lobby_button_update_icon(atom/movable/screen/lobby/button/sign_up/lobby_button, location, control, params, mob/dead/new_player/user)
+	var/mob/player = lobby_button.get_mob()
+	var/skub_stance = skubbers[player.ckey]
+	switch(skub_stance)
+		if(TRUE)
+			lobby_button.base_icon_state = "signup_on"
+		if(FALSE)
+			lobby_button.base_icon_state = "signup"
```

Comment:
id prefer these to be their own defines, skub stance = true doesnt really say anything